docs/coding/data-modeling.md

```diff
@@ -95,19 +95,28 @@ to avoid loss of precision due to floating-point approximations.
 When the multiplier is a power of 10 (e.g. `10 ^ n`), then the exponent `n` is referred to as an
 _asset scale_. For example, representing USD in cents uses an asset scale of `2`.
 
+The 128-bit representation defines the precision, but not the scale.
+
 #### Examples
 
-- In USD, `$1` = `100` cents. So for example,
+- In USD, `$1` = `100` cents. Using an asset scale of `2`;
   - The fractional amount `$0.45` is represented as the integer `45`.
   - The fractional amount `$123.00` is represented as the integer `12300`.
   - The fractional amount `$123.45` is represented as the integer `12345`.
 
-### Oversized Amounts
+- In JPY, `¥1` = `1` yen. Using an asset scale of `0`;
+  - The fractional amount `¥123` is represented as the integer `123`.
```

Comment:
Maybe we should have 3 examples of each, to keep the symmetry going between each example.